pages/posts/analysis-of-json-schema-defects.md

```diff
@@ -0,0 +1,113 @@
+---
+title: "An Analysis of JSON Schema Defects"
+date: 2023-09-10
+tags:
+  - Specification
+type: Opinion
+cover: /img/posts/2023/analysis-of-json-schema-defects/cover.webp
+authors:
+  - name: Fabien Coelho
+    photo: /img/avatars/fabien.jpg
+    link: https://www.linkedin.com/in/fabien-coelho-65433a18/
+    byline: Professor in Computer Science
+  - name: Claire Medrala
+    photo: /img/avatars/claire.jpg
+    link: https://www.linkedin.com/in/claire-medrala/
+    byline: Research Engineer
+excerpt: Evidences show that schemas are hard to write, and suggest changes in the spec
```

Comment:
I absolutely object to putting (or hinting at) third party recommendations for spec changes in our blog.

This is the JSON Schema blog.  It is a place for us to show off what it can do, not a forum for discussion about change or shortcomings.  The appropriate place for that is [issues](https://github.com/json-schema-org/json-schema-spec/issues) and [discussions](https://github.com/orgs/json-schema-org/discussions).

Reply:
> I absolutely object to putting (or hinting at) third party recommendations for spec changes in our blog.

Ok. The blog is only for *your* recommendations and discussions. Fine, it is your blog after all. As an academic, we are more used to open discussions and disagreements.

> This is the JSON Schema blog. It is a place for us to show off what it can do, not a forum for discussion about change or shortcomings. The appropriate place for that is [issues](https://github.com/json-schema-org/json-schema-spec/issues) and [discussions](https://github.com/orgs/json-schema-org/discussions).

Ok.